test/functional/wallet_avoidreuse.py

```diff
@@ -193,7 +198,7 @@ def test_fund_send_fund_send(self):
         '''
         self.log.info("Test fund send fund send")
 
-        fundaddr = self.nodes[1].getnewaddress()
+        fundaddr = self.nodes[1].getnewaddress(label="", address_type="legacy")
```

Comment:
nit: `label` isn't needed

Reply:
It would be, once address type default is changed, no?